lib/pages/home_page.dart

```diff
@@ -209,15 +230,45 @@ class _HomePageState extends State<HomePage> {
     final heading = _heading(context);
     final previousScoreHolder = _previouScoreHolder(context);
     final thisWeekHolder = _thisWeekHolder(context);
+    final warningBanner = MaterialBanner(
+      leading: Icon(
+        Icons.warning,
+        color: Colors.red,
+      ),
+      content:
+          const Text('No pedometer available. Functionality will be limited.'),
+      actions: [
+        TextButton(
+          child: Text('Ok'),
+          onPressed: () => setState(() => pedometerWarn = false),
+        )
+      ],
+    );
+    final appBar = AppBar(
+      title: heading,
+      centerTitle: true,
+      backgroundColor: Theme.of(context).scaffoldBackgroundColor,
+      actions: [
+        IconButton(
+          onPressed: () => launch(
+              URL_USER_MANUAL),
+          icon: Icon(Icons.help_outline),
+          color: Colors.blue,
+        )
+      ],
+    );
 
     return Scaffold(
+        appBar: appBar,
         body: SafeArea(
             child: Column(
           mainAxisAlignment: MainAxisAlignment.spaceEvenly,
           mainAxisSize: MainAxisSize.min,
           children: [
-            heading,
-            Divider(),
+            Visibility(
+              visible: pedometerWarn == true,
+              child: warningBanner,
+            ),
             previousScoreHolder,
```

Comment:
Could you add some padding around the previousScoreHolder?  Like Padding(
                padding: EdgeInsets.fromLTRB(0, 20, 0, 20),
                child: previousScoreHolder),